ReSharper.FSharp/src/Daemon.FSharp/src/FSharpSymbolHighlightingUtil.cs

```diff
@@ -82,6 +98,13 @@ public static string GetHighlightingAttributeId([NotNull] this FSharpSymbol symb
         case FSharpEntity entity when !entity.IsUnresolved:
           return GetEntityHighlightingAttributeId(entity);
 
+        case FSharpParameter parameter: // FSharpParameter is a static member in a generic constraint, not a method parameter
+          if (parameter.FullName.Equals("new"))
```

Comment:
Why are you using `.Equals` instead of just `=`? Less noise for my taste.

Reply:
Thanks. But this code was removed